### PR TITLE
Homework4 PR

### PR DESCRIPTION
**Pull request by tarasandrusiv**

### CODE REVIEW

**Comment by tarasandrusiv** [comment]
+ minor fix for homework#3

File: homework4.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+}
+
+for letter, number in vowel_count.items():
+    print(f"| {letter:^8}| {number:^7} |")
+
+print("-" * sep_num)
+print("See modified poem text below")
+print("-" * sep_num)
+
+rep_table = str.maketrans('aAeEiIoOuU', 'àÀéÉíÍóÓúÚ')
```
Comment (antohaUa) [inline homework4.py:80]: Yes. The best way for such kind of replacement

File: homework4.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
+"""
+Homework #4.
+Flow:
```
Comment (antohaUa) [inline homework4.py:3]: The best choice:  docstring should have Caption + full task description + Flow ( Solution algorithm).
In you case missed full task description

**Review by antohaUa — CHANGES_REQUESTED** [review]
Fixes required

**Pull request merged**